requirements/common_gpu.txt

```diff
@@ -1,2 +1,3 @@
 nvgpu; sys_platform != 'win32'
 nvgpu==0.10.0; sys_platform == 'win32'
+ninja==1.11.1.1
```

Comment:
is ninja needed to run an inference or to export? If export only we shouldn't add this as a dependency

Reply:
This is needed for loading the .so file. I can add this in requirements.txt for the example?

Reply:
This doesn't work.